game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/MainPanel.java

```diff
@@ -38,20 +38,19 @@
   private static final Dimension initialSize = new Dimension(800, 620);
 
   private final JButton playButton =
-      JButtonBuilder.builder()
+      new JButtonBuilder()
           .title("Play")
           .toolTip(
               "<html>Start your game! <br>"
                   + "If not enabled, then you must select a way to play your game first: <br>"
                   + "Play Online, or Local Game, or PBEM, or Host Networked.</html>")
           .build();
-  private final JButton cancelButton = JButtonBuilder.builder().title("Cancel").build();
+  private final JButton cancelButton = new JButtonBuilder().title("Cancel").build();
 
-  private final JPanel gameSetupPanelHolder = JPanelBuilder.builder().borderLayout().build();
+  private final JPanel gameSetupPanelHolder = new JPanelBuilder().borderLayout().build();
   private final JPanel mainPanel;
   private final JSplitPane chatSplit;
-  private final JPanel chatPanelHolder =
-      JPanelBuilder.builder().borderLayout().preferredHeight(62).build();
+  private final JPanel chatPanelHolder = new JPanelBuilder().preferredHeight(62).build();
```

Comment:
What happened to borderLayout?

Reply:
Looks like it just got dropped. Added back in: 3aaff4467